conans/tools.py

```diff
@@ -610,7 +611,7 @@ def __init__(self, runner=None, os_info=None, tool=None):
         self._is_up_to_date = False
         self._tool = tool or self._create_tool(os_info)
         self._tool._sudo_str = "sudo " if self._sudo else ""
-        self._tool._runner = runner or ConanRunner()
+        self._tool._runner = runner or ConanRunner(print_commands_to_output=True)
```

Comment:
As I tried to explain in the PR description, I think the current way of testing for a stream ouput (not the exit code) with `test -n "$(cmd)"` for a system package tool command is not working. 
With this change I wanted to get the stdout from the command I was running to parse it and decide whether it was successful or not.